package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "api-portfolio-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "example-data.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/env-cmd ./node_modules/.bin/nodemon index.js",
+    "dev:migrate": "./node_modules/.bin/env-cmd ./node_modules/.bin/sequelize db:migrate --env development --config ./config/sequelize.js",
+    "lint": "npm run lint:pug && npm run lint:eslint",
+    "lint:pug": "./node_modules/.bin/pug-lint . ",
+    "lint:eslint": "./node_modules/.bin/eslint --format codeframe .",
+    "test": "./node_modules/.bin/env-cmd  ./node_modules/.bin/mocha tests/**/*.test.js"
```

Comment:
Intermittent failing tests which is happening because you are allowing it to actually connect to the database.

![image](https://user-images.githubusercontent.com/5775716/83889321-03eb6180-a719-11ea-82f5-e8b43e7920f6.png)

![image](https://user-images.githubusercontent.com/5775716/83890852-537e5d00-a71a-11ea-9a1d-79554b4ce940.png)